packages/superinput/README.md

```diff
@@ -0,0 +1,121 @@
+# Super Input
+
+VsCode-like autocomplete for input boxes. Uses Codemirror6 under the hood.
+
+## Usage & Commands
+
+---
+
+### Importing in Studio
+
+1. Install `@botpress/superinput` (only in studio for now)
+   - `yarn workspace @botpress/studio-ui add @botpress/superinput`
+   - this set may already be done, check `package.json` for `@botpress/superinput`
+2. Import component
+
+   ```jsx
+   import { useState } from 'react'
+   import { Superinput } from '@botpress/superinput'
+
+   const SomeComponent = () => {
+     const [value, setValue] = useState()
+
+     return <Superinput value={value} onChange={setValue} />
+   }
+   ```
+
+3. Import different type
+
+   ```jsx
+   import { Superinput, SI_TYPES } from '@botpress/superinput'
+   ...
+     return <Superinput type={SI_TYPES.EXPRESSION} value={value} onChange={setValue} />
+
+   ```
+
+### Building `doctree`
+
+Building the doctree generates a JSON file that exports documentation from `botpress.d.ts`.
+
+```shell
+yarn workspace @botpress/superinput doctree
+```
+
+### Run Demo environment
+
+After building the `doctree` (see above), you can spin up the dev environment to test it out
+
+```shell
+yarn workspace @botpress/superinput demo
+```
+
+### Build package
+
+```shell
+yarn workspace @botpress/superinput build
+```
+
+### Watch build
+
+Does not create `doctree` for you, must be generated separately
+
+```shell
+yarn workspace @botpress/superinput dev
+```
+
+## Props
+
+---
+
+```typescript
+enum SI_TYPES { // types of superinput
+  TEMPLATE = 0, // for template strings using the {{}} delimiter
+  EXPRESSION = 1, // for full javascript expressions that evaluate its full output
+  BOOL = 2 // for full javascript expressions that evaluate its output to a bool
+}
```

Comment:
Could we use text intead of numbers? it's easier to use (also, please avoid the prefix "I" for interfaces, we have typings, its not necessary)
```js
type SuperInputType = 'template' | 'expression' | 'bool' 

interface SuperInputProps {
  type: SuperInputType
}
```